CMakeLists.txt

```diff
@@ -16,6 +16,14 @@ endif (APPLE)
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/)
 
+if (ANDROID)
+    set(CMAKE_FIND_ROOT_PATH ${OPENMW_DEPENDENCIES_DIR} "${CMAKE_FIND_ROOT_PATH}")
+    set (OSG_PLUGINS_DIR CACHE STRING "")
+    if (NOT FREETYPE_LIBRARY_REL AND NOT FREETYPE_LIBRARY_DBG)
+        message(FATAL_ERROR "Freetype required for Android, but not found! Without freetype, fonts will not work !")
+    endif()
```

Comment:
Move this inside FindMyGUI.cmake e.g. by 

<pre>
find_package(freetype REQUIRED)
</pre>